### ISSUE TITLE
XML Formatter removed from scss-lint

### ISSUE THREAD

**Issue by iamdarrenhall** [juanfran/gulp-scss-lint#32]
A [recent commit](https://github.com/brigade/scss-lint/commit/c6df8d7940d403d3fa4e315a6b93ef5dd3141f14) in scss-lint has removed the XML formatter, which in turn breaks gulp-scss-lint as `--format=XML` is now an invalid output format.

```
[21:32:41] Error: Error in plugin 'gulp-scss-lint'
Message:
    Command line usage error
```
```
Invalid output format specified: XML
```

**Comment by JohnAlbin** [comment]
ugh. Experiencing this now.

**Comment by JohnAlbin** [comment]
And… this is fixed by using 0.2.0. :-) Nice work!

Changed:
```
"gulp-scss-lint": "^0.1.12",
```
to:
```
"gulp-scss-lint": "^0.2.0",
```

**Comment by JohnAlbin** [comment]
@juanfran++
